### ISSUE TITLE
Documentation: How to install

### ISSUE THREAD

**Issue by Andro-Meta** [SebastianAigner/promptK#1]
It says this is available for Android, but I don't see any instructions or links to install it on Android.

**Comment by tscholze** [comment]
@Andro-Meta which IDE do you use? :)

**Comment by Andro-Meta** [comment]
> @Andro-Meta which IDE do you use? :)

What IDE do you want me to use to install this on android?

**Comment by tscholze** [comment]
@Andro-Meta I think the only way to install it, is to build the app locally using an IDE and deploying it to your phone via Android Studio.

**Comment by SebastianAigner** [comment]
Yes, as of now, you can use Fleet or Android Studio to build and deploy the application to your phone @Andro-Meta . However, it seems we should probably spend one of our live streams setting up some sort of deployment or, at the very least, APK generation so that people can try it out. I've added it to the TODO in our readme!
